environments/common/layouts/everything

```diff
@@ -81,4 +81,8 @@ openhpc
 
 [ansible_init:children]
 # Hosts to run ansible-init
-cluster
\ No newline at end of file
+cluster
+
+[k3s:children]
+# Hosts to run k3s server/agent
+cluster
```

Comment:
Lets change this to `openhpc`. There are cases where the "cluster" (i.e. hosts controlled by this ansible) is wider e.g. extra hosts added to do ... stuff. Bit of a subtle argument TBH.